chapter06/mssw/file_reader.pm

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/perl
```

Comment:
The lack of a package means this is really just a script with a .pm files extension, that is executed inline by the require statement. We won't need to worry about packages until chapter 10 though, so you'll see the difference then.